### PR TITLE
Update loosely_coupled_architecture.md

### PR DESCRIPTION
**Pull request by reddymahendra52**
Fixes issue #142

References used: 
1. https://medium.com/@marciosete/loosely-coupled-architecture-6a2b06082316
2. https://searchapparchitecture.techtarget.com/tip/Can-a-loosely-coupled-architecture-reduce-technical-debt

### CODE REVIEW

**Comment by reddymahendra52** [comment]
Proposed changes for the style guide.

**Current template:** 
Current_snip

**Proposed change:**
Change

Since we are mostly following the latter one that's why updated the style guide.

File: definitions/loosely_coupled_architecture.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 
-When applications and processes are reliant on each other, the development lifecycle also needs to be intertwined. Updates, scaling, testing, and new features will be implemented against more extensive codebase. The larger the codebase and its dependencies, the more the application can be considered a monolith.
+### Problem it addresses
 
-By decoupling an application, application development will be quicker since external dependencies will not hamper it. At an organizational level, teams can be structured around their competency, focusing on the specific application.
+When applications and processes are reliant on each other, the development lifecycle also needs to be intertwined. Updating, scaling, testing, and adding new features against an extensive codebase increases technical debt and decreases feasibility. The larger the codebase and its dependencies, becomes more or less like Tight Coupled Architectures (aka Monoliths), which are brittle, hard to change, test, and release.
```
Comment (JasonMorgan) [inline definitions/loosely_coupled_architecture.md:14]: This last sentence, "The larger the codebase and its dependencies, becomes more or less like Tight Coupled Architectures (aka Monoliths), which are brittle, hard to change, test, and release." Doesn't make sense. It looks like it's two somewhat related sentences jammed together.

Comment (reddymahendra52) [reply definitions/loosely_coupled_architecture.md:14]: Okay,
Thanks for the suggestion.
I will remove the last line.

**Comment by reddymahendra52** [comment]
Suggested changes has been done.

**Review by JasonMorgan — APPROVED** [review]

**Review by CathPag — APPROVED** [review]
LGTM

**Pull request merged**